.docker/remove-volumes.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+source .docker/.external-volumes.rc
+
+for volume in "${EXTERNAL_VOLUMES[@]}"; do
+    # if the volume is existing, grep matches it in the 1-item list
+    if docker volume ls --quiet --filter name="${volume}" | grep "${volume}"; then
```

Comment:
```suggestion
    if [ $(docker volume ls --quiet --filter name="${volume}") == "${volume}" ]; then
```
Perhaps this is more explicit?